tests/drivers/build_all/gpio/app.overlay

```diff
@@ -440,6 +441,28 @@
 				ngpios = <8>;
 				#gpio-cells = <2>;
 			};
+
+			test_spi_max14906: max14906@5 {
+				compatible = "adi,max1906-gpio";
+				status = "okay";
+				reg = <0x06>;
+				spi-max-frequency = <0>;
+				gpio-controller;
+				#gpio-cells = <2>;
+				ngpios = <4>;
+				crc-en;
+				spi-addr = <0>;
+				ow-en = <0 0 0 0>;
+				vdd-ov-en = <0 0 0 0>;
+				gdrv-en = <0 0 0 0>;
+				sh-vdd-en = <0 0 0 0>;
+				drdy-gpios = <&test_gpio 0 0>;
+				fault-gpios = <&test_gpio 0 0>;
+				sync-gpios = <&test_gpio 0 0>;
+				en-gpios = <&test_gpio 0 0>;
+			};
+
+>>>>>>> 60f39fa91c5 (drivers: gpio: Add MAX14906 industrial input/output)
```

Comment:
Remove

Reply:
@MaureenHelm this line it is not present in current patch version

Reply:
Nice catch. Thanks !
Fixed.

I am wondering : shouldn't checkpatch catch it ?

Reply:
It should, I'm not sure why it didn't catch this.